operate/webapp/src/main/java/io/camunda/operate/webapp/security/permission/PermissionsConfigurer.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.webapp.security.permission;
+
+import io.camunda.security.configuration.SecurityConfiguration;
+import io.camunda.security.impl.AuthorizationChecker;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class PermissionsConfigurer {
```

Comment:
```suggestion
public class SecurityConfigurer {
```
As this config class also creates `TenantsService` bean

Reply:
Isn't tenant also used to control permissions after all ?